src/vcpkg/paragraphs.cpp

```diff
@@ -420,7 +420,7 @@ namespace vcpkg::Paragraphs
         else
         {
             vcpkg::Checks::check_exit(VCPKG_LINE_INFO,
-                                      !fs.exists(control_path, IgnoreErrors{}),
+                                      !(fs.exists(control_path, IgnoreErrors{}) && fs.is_regular_file(control_path)),
```

Comment:
```suggestion
                                      !fs.is_regular_file(control_path),
```

is_regular_file implies that it exists.